lib/MusicBrainz/Server/Controller/Statistics.pm

```diff
@@ -435,7 +435,7 @@ sub edits : Path('edits') {
     my %by_category;
     for my $class (EditRegistry->get_all_classes) {
         $by_category{$class->edit_category} ||= [];
-        push @{ $by_category{$class->edit_category} }, {edit_type => $class->edit_type, edit_name => $class->edit_name};
+        push @{ $by_category{$class->edit_category} }, {edit_type => $class->edit_type, edit_name => $class->l_edit_name};
```

Comment:
Shouldn’t the property be renamed to `l_edit_name` as well?
Or should `edit_type_name_context` be additionally passed instead?

Reply:
Renaming it to `l_edit_name` sounds sensible at least.

Reply:
I didn't rename it because we didn't do that for `Edit_Types` where we were using `l_` already, but it seems fine to rename both for clarity, so that we don't later think we forgot to translate it when looking at the code. Will do.